tests/integration/store/test_version_store_corruption.py

```diff
@@ -24,8 +24,8 @@ def n_append(library, library_name, total_appends, rows_per_append, bulk_data_ts
     return open_last_row
 
 
-def _corrupt_with_append_only(library, library_name):
-    def do_fail(version):
+def _corrupt_with_append_only(library, _library_name):
```

Comment:
The `_` prefix on `library_name` will mean it doesn't match the pytest fixture, no?